### DISCUSSION TITLE
Possibility of running the incompleted program

### DISCUSSION THREAD

**Discussion by milad7093** [connectomicslab/connectomemapper3#207] [Q&A] [answered]
Dear Sebastian 

I ran my program for ten subjects on my computer ( linux ubuntu 20.04  with 24 cores and 32 GB physical memory (RAM)) . After completing the anatomical pipline, The program performed all diffusion steps except for "connectome_stage.compute_matrice" ( see the attached file).  Is there any way to re-run the program's code for computing connectome metrics at this step  without the need to run it from the first? 

With kind regards

Milad

[sub-3105_ses-1_log.txt](https://github.com/connectomicslab/connectomemapper3/files/9128371/sub-3105_ses-1_log.txt)

**Comment by sebastientourbier** [comment, marked as answer]
Dear Milad,

If you did not delete the `derivatives/nipype-1.8.0`, which stores all intermediate nipype outputs, the pipeline should be able to collect the precomputed outputs of all finished and unchanged interface nodes and resume to `compute_matrice`.

**Comment by milad7093** [comment]
Dear Sebastian 

Many thanks to your help. The program ran completely.

**Comment by sebastientourbier** [comment]
Great! You're welcome :)
